akka-cluster/src/main/scala/akka/cluster/ClusterDaemon.scala

```diff
@@ -428,8 +428,8 @@ private[cluster] class ClusterCoreDaemon(publisher: ActorRef, joinConfigCompatCh
     }
 
     if (seedNodes.isEmpty) {
-      if (isClusterBootstrapUsed)
-        logDebug("Cluster Bootstrap is used for joining")
+      if (isClusterBootstrapAvailable)
+        logInfo("No seed nodes found in configuration, relying on Cluster Bootstrap for joining")
```

Comment:
Shall we add something like "ensure Cluster Bootstrap is being started by your application code"?

Reply:
I like the suggestion, but I'm not sure how to word it clearly. I'll merge as-is for now, can always improve further later.